vhdbuilder/packer/components.json

```diff
@@ -95,10 +95,11 @@
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-npm:*",
       "amd64OnlyVersions": [
-        "v1.4.9",
-        "v1.4.12"
```

Comment:
1.4.12 is not in use?

Reply:
No. just before holidays we found an issue in it, so aborted release process.